lib/config.ts

```diff
@@ -67,27 +69,27 @@ function createConfig ({
     configFilepath: path.resolve(cliArgs.config || CONFIG_DEFAULT),
     dynamicEntryFilepath: path.join(merged.output, OUTPUT_DYNAMIC_PAGES_ENTRY),
     emitter: createEmitter()
-  }
+  } as config
 
-  debug('config created', global.__presta__)
+  debug('config created', (global as any).__presta__)
 
-  return global.__presta__
+  return (global as any).__presta__
 }
 
-function removeConfigValues () {
-  global.__presta__ = createConfig({
-    ...global.__presta__,
+export const removeConfigValues = () => {
+  (global as any).__presta__ = createConfig({
```

Comment:
I bet we could write this in a way that doesn't require all these type casts lol, I was just lazy when refactoring this a few weeks ago 😛

Reply:
yeah was gonna suggest augmenting the window interface to avoid the assertions i.e.

```ts
declare global {
  interface Window {
    __presta__: ...;
  }
}
```

Reply:
Definitely. Just wanted to get something working asap.

Reply:
Totally fair, I left this in a less-than-ideal state haha. @achou11 good call, might be the move here.